docs/cookbooks-retention-expiry.md

```diff
@@ -473,3 +479,6 @@ The diagram below illustrates one of the cases that the consumed storage size is
 ![](/assets/retention-storage-size.svg)
 
 If you do not have any retention period and you never have much of a backlog, the upper limit for retained messages, which are acknowledged, equals the Pulsar segment rollover period + entry log rollover period + (garbage collection interval * garbage collection ratios).
+
+[backlogquotacheckintervalinseconds]: https://pulsar.apache.org/reference/#/next/config/reference-configuration-broker?id=backlogquotacheckintervalinseconds
+[precisetimebasedbacklogquotacheck]: https://pulsar.apache.org/reference/#/next/config/reference-configuration-broker?id=precisetimebasedbacklogquotacheck
```

Comment:
1. What's the context of these two paras? Do you intend mean this?

```suggestion
To configure checks, you can use `backlogquotacheckintervalinseconds` and 
`precisetimebasedbacklogquotacheck` parameters. Details see **Configuration > Plusar > Broker** on [Pulsar Reference Site](https://pulsar.apache.org/reference).
```

2. Suggest using general link (https://pulsar.apache.org/reference) instead of specific links to reduce maintenance costs and potential errors. We've applied this strategy across all docs.

Reply:
[This comment](https://github.com/apache/pulsar-site/pull/412#discussion_r1113751246) was left two weeks ago. The reason for my suggestion (as a temporary workaround) at that time was:

For example, if we want to publish a new next code release (2.12.0) with docs, the whole 2.12.x doc set is copied from `master`. If we use `https://pulsar.apache.org/reference/#/next/xxxxxxxx` in links, then the links in the old doc set will always point to the latest Reference site, which is incorrect.

 < < < < < < < < < < < < < < < <

But now you can write it as `[precisetimebasedbacklogquotacheck]: https://pulsar.apache.org/reference/#/@pulsar:version_origin@/config/reference-configuration-broker?id=precisetimebasedbacklogquotacheck` since https://github.com/apache/pulsar-site/pull/456 was merged yesterday.